src/main/java/net/minecraftforge/event/ForgeEventFactory.java

```diff
@@ -636,9 +637,9 @@ public static void onChunkPopulate(boolean pre, IChunkGenerator gen, World world
         MinecraftForge.EVENT_BUS.post(pre ? new PopulateChunkEvent.Pre(gen, world, rand, x, z, hasVillageGenerated) : new PopulateChunkEvent.Post(gen, world, rand, x, z, hasVillageGenerated));
     }
 
-    public static LootTable loadLootTable(ResourceLocation name, LootTable table)
+    public static LootTable loadLootTable(ResourceLocation name, LootTable table, LootTableManager lootTableManager)
```

Comment:
This is a breaking change, please keep the old constructor and deprecate it instead of removing it.